pkg/route/graph/edges.go

```diff
@@ -0,0 +1,30 @@
+package graph
+
+import (
+	"github.com/gonum/graph"
+
+	osgraph "github.com/openshift/origin/pkg/api/graph"
+	kubeedges "github.com/openshift/origin/pkg/api/kubegraph"
+	kubegraph "github.com/openshift/origin/pkg/api/kubegraph/nodes"
+	routegraph "github.com/openshift/origin/pkg/route/graph/nodes"
+)
+
+// AddRouteEdges adds an edge that connect a service to a route in the given graph
+func AddRouteEdges(g osgraph.MutableUniqueGraph, node *routegraph.RouteNode) {
+	for _, n := range g.(graph.Graph).Nodes() {
+		if serviceNode, ok := n.(*kubegraph.ServiceNode); ok {
+			if RoutesTo(node.Route, serviceNode.Service) {
+				g.AddEdge(node, serviceNode, kubeedges.ExposedThroughServiceEdgeKind)
```

Comment:
This isn't the correct edge type.  A `Route` could expose a service, but `kubeedges.ExposedThroughServiceEdgeKind` is a reverse link about things that a service exposes.  Not things that expose services.